test/test_mt940.rb

```diff
@@ -9,6 +17,8 @@ class TestMt940 < Test::Unit::TestCase
   def test_it_should_parse_fixture_files_correctly
     Dir[File.dirname(__FILE__) + "/fixtures/*.txt"].reject { |f| f =~ /sepa_snippet/ }.each do |file|
       data = MT940.parse(IO.read(file))
+      data.flatten.each { |field| field.extend(Mt940FieldEncoderWithoutContent) }
```

Comment:
I'm using this slightly odd approach to avoid the pain of going through `sepa_mt9401.yml` and updating every single one of the many, many entries with the content. This functionality is so basic that it is not worth testing in that way, so mixing in `Mt940FieldEncoderWithoutContent` allows us to avoid that.